pallets/asset/src/lib.rs

```diff
@@ -1240,29 +1240,29 @@ decl_module! {
             Ok(())
         }
 
-        /// Sets the treasury DID to None. The caller must be the asset issuer. The asset
-        /// issuer can always update the treasury DID using `transfer_treasury`. If the issuer
-        /// clears their treasury DID then it will be immovable until either they transfer
-        /// the treasury DID to an actual DID, or they add a claim to allow that DID to move the
+        /// Sets the primary issuance agent to None. The caller must be the asset issuer. The asset
+        /// issuer can always update the primary issuance agent using `transfer_primary_issuance_agent`. If the issuer
+        /// clears their primary issuance agent then it will be immovable until either they transfer
+        /// the primary issuance agent to an actual DID, or they add a claim to allow that DID to move the
         /// asset.
         ///
         /// # Arguments
         /// * `origin` - The asset issuer.
         /// * `ticker` - Ticker symbol of the asset.
         #[weight = 250_000]
-        pub fn clear_primary_issuance_did(
+        pub fn clear_primary_issuance_agent(
```

Comment:
It better to have `remove` prefix instead of `clear` prefix to have consistency with the other code parts